pkg/chunk/azure/blob_storage_client.go

```diff
@@ -214,3 +229,33 @@ func (b *BlobStorage) DeleteObject(ctx context.Context, blobID string) error {
 func (b *BlobStorage) PathSeparator() string {
 	return b.delimiter
 }
+
+func (b *BlobStorage) selectBlobUrlFmt() string {
+	switch b.cfg.CloudEnvironment {
+	case "AzureGlobal":
```

Comment:
Could you create constants for these `AzureGlobal`, `AzureChinaCloud`, etc please?

Reply:
done